src/network/graphQuery.js

```diff
@@ -127,14 +125,20 @@ class GraphQuery {
 * @param filters {Array} Array of objects for filtering. ie. [{ status: 'WAITRESULT' }, { status: 'OPENRESULTSET' }]
 * @param orderBy {Object} Object with order by fields. ie. { field: 'blockNum', direction: 'ASC' }
 */
-export function queryAllTopics(filters, orderBy) {
+export function queryAllTopics(filters, orderBy, limit, skip) {
   const request = new GraphQuery('allTopics', TYPE.topic);
   if (!_.isEmpty(filters)) {
     request.setFilters(filters);
   }
   if (!_.isEmpty(orderBy)) {
     request.setOrderBy(orderBy);
   }
+  if (!_.isNil(skip) && limit >= 0) {
```

Comment:
`limit`, not skip

Reply:
use `_.isFinite(limit) && limit > 0`.
it doesn't make sense to set the limit as 0 since there won't be any items returned.